cluster/ubuntu/build.sh

```diff
@@ -26,12 +26,15 @@ function cleanup {
 }
 trap cleanup SIGHUP SIGINT SIGTERM
 
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
+pushd ${KUBE_ROOT}/cluster/ubuntu
```

Comment:
This is still a great imporvement since i can call ./cluster/ubuntu/build.sh from anywhere now.